final_prep/annotations.rb

```diff
@@ -3,41 +3,72 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# method initiated called build_a_bear including 5 parameters (name, age, fur, clothes, special_power)
 def build_a_bear(name, age, fur, clothes, special_power)
+#Declare a local variable. Assign local variable to string with name paramteter
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #Declare local variable "demographics" with array containing two parameters "name and age"
   demographics = [name, age]
+  #Local variable declared "power_saying" with string value incorporating/interpolating special_power inside
   power_saying = "Did you know that I can #{special_power}?"
+  # hash created for built_bear
   built_bear = {
+    #assign "basic info" to key value of "demographics"
     'basic_info' => demographics,
+    #assign value "clothes" to key "clothes"
     'clothes' => clothes,
+    #assign key "exterior" to value "fur"
```

Comment:
What are these values? Where do they come from? If a value is a variable, it can be useful to track where it comes from, like is it passed as a parameter, or was it declared previously.